app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb

```diff
@@ -155,14 +128,21 @@ app.controller('RecipeFinderCtrl', ['$scope', '$http', '$routeParams', function(
     }
   };
 
+  $scope.searchWord = "Ingredient";
+
   $scope.addKeyword = function() {
     if ($scope.newKeyword != undefined && $scope.newKeyword != "" && $scope.newKeyword.length > 2) {
       if (checkDuplicates($scope.newKeyword,$scope.search_parameters.keywords) == true) {
```

Comment:
If `checkDuplicates($scope.newKeyword,$scope.search_parameters.keywords)` already returns `true`, then your condition is basically `if true == true`. In this case, you don't need `== true`.